src/main/globalSetting.js

```diff
@@ -1,11 +1,8 @@
-/**
- * Set `__static` path to static files in production
- * https://simulatedgreg.gitbooks.io/electron-vue/content/en/using-static-assets.html
- */
-
+// Set `__static` path to static files in production
 if (process.env.NODE_ENV !== 'development') {
   global.__static = require('path').join(__dirname, '/static').replace(/\\/g, '\\\\')
 }
 
 global.MARKTEXT_DEBUG = process.env.MARKTEXT_DEBUG || process.env.NODE_ENV !== 'production'
+global.MARKTEXT_DEBUG_VERBOSE = global.MARKTEXT_DEBUG && process.env.MARKTEXT_DEBUG_VERBOSE
```

Comment:
Why add `MARKTEXT_DEBUG_VERBOSE `if we already have `MARKTEXT_DEBUG `?

Reply:
To be more verbose and prevent spam in debug mode.